pandora-common/src/gameLogic/interactions/_interactionConfig.ts

```diff
@@ -36,12 +42,30 @@ export const INTERACTION_CONFIG = {
 			allowOthers: true,
 		},
 	},
+	assetPrefFavorite: {
+		visibleName: 'Add and remove assets with favorite preference',
+		defaultPermissions: {
+			allowOthers: true,
+		},
+	},
+	assetPrefNormal: {
+		visibleName: 'Add and remove assets with normal preference',
+		defaultPermissions: {
+			allowOthers: true,
+		},
+	},
+	assetPrefMaybe: {
+		visibleName: 'Add and remove assets with maybe preference',
+		defaultPermissions: {
+			allowOthers: true,
+		},
+	},
```

Comment:
I'm not sure about these being part of interactions config... I would rather move them to be their own category. If you wouldn't mind I can implement that as I have a good idea how to do that. (I intended for the solution to be used that way, but there is so far no example of how I actually imagined it being done)